### ISSUE TITLE
[book-dp1] Text and rendering discrepancies in Common Symbols table

### ISSUE THREAD

**Issue by Chihiro2000GitHub** [QuantEcon/book-dp-public#24]
Hi @mmcky @jstac 

I'd be grateful if you could check the following when you have time. These might be minor differences, but I wanted to flag them just in case.

## Description

While comparing the `Common Symbols` table in the PDF against the MyST/HTML edition, I noticed a few small text and rendering discrepancies.

## Discrepancies

Compared with the PDF:

* In the first row, the indicator function symbol appears to be rendered differently.

  * PDF: the indicator uses a double-struck `1`.
  * MyST/HTML: the indicator appears to use a regular `1`.

* In the row for the natural, integer, real, and complex numbers, the symbol itself differs.

  * PDF symbol: `N, Z, R, and C`
  * MyST/HTML symbol: `N, Z, R, C`

* In the same row, the meaning text also differs slightly.

  * PDF: “the natural, integer, real, and complex numbers respectively”
  * MyST/HTML: “the natural, integer, real, and complex numbers”

* The row for `Z_+, R_+, etc.` differs slightly.

  * PDF uses `etc.`
  * MyST/HTML uses `...`

* The symbols for `D(F)`, `L(X)`, and `M(X)` appear to be rendered in a slightly different style from the PDF. This may just be a font/rendering difference.

* In the row for `M(X)`, the cross-reference at the end of the meaning text appears to be missing in the MyST/HTML version.

  * PDF: “the set of Markov operators in `L(X)` (see §2.3.3.4)”
  * MyST/HTML: “the set of Markov operators in `L(X)`”

* The abbreviation for independent and identically distributed is capitalized differently.

  * PDF: `IID`
  * MyST/HTML: `iid`

**Comment by jstac** [comment]
Thanks @Chihiro2000GitHub, great catches!

@mmcky I wonder if \mathbb 1 not rendering is a mathjax issue.  I seem to remember we've had this problem with html before.

It would be good to get all the fonts lined up --- for example, mathscr L vs mathcal L might be rendering differently in the pdf and html.

The iid issue might be due to the use of \sc (small caps).

**Comment by mmcky** [comment]
thanks @jstac -- one of the tricky elements of this conversation process is LaTeX + KaTeX (used by mystmd!) another dimension on top of mathjax.

**Comment by jstac** [comment]
Ah, another layer of complexity :-)

Thanks @mmcky

**Comment by mmcky** [comment]
Thanks @Chihiro2000GitHub — confirmed all of these against `book/common_symbols.tex`. Triaged and tracked in two issues:

**Rendering / config (book-dp1):**
- **Indicator `𝟙` → plain `1`** — @jstac's hunch is close: the web renderer here is KaTeX (not MathJax), and **KaTeX can't render blackboard-bold *digits*** (`\mathbb{1}`), so it falls back to `1`. Fix is a macro remap in `myst.yml`.
- **`D(F)` / `L(X)` / `M(X)` font style** — both PDF and web map to `\mathscr`; KaTeX's script glyphs just differ from LaTeX's (the mathscr-vs-mathcal point @jstac raised). Cosmetic; will review.

**Content fidelity (book-dp1):**
- **Missing `M(X)` cross-ref** "(see §2.3.3.4)" — will restore as a `{ref}` to the Positive/Markov Operators section.
- **"and" / "respectively" / `etc.` vs `...`** — these are in the source; will align the web table with the PDF wording.
- **`IID` vs `iid`** — @jstac is right, the source uses small caps (`{\sc iid}`); will render it as `IID`.

**Converter (claude-latex-to-myst):** the `iid` small-caps loss and the dropped table cross-ref are general LaTeX→MyST gaps (`{\sc ...}` declaration form and `\ref` inside `tabular` cells), so they're also tracked upstream so other tables benefit.

Tracking:
- book-dp1 fixes → QuantEcon/book-dp1#351
- converter hardening → QuantEcon/claude-latex-to-myst#107

Will update here once the table fixes land on the site.

**Comment by mmcky** [comment]
Status update on this one: the **general converter gaps** your report surfaced are now fixed and deployed — `{\sc …}` small-caps now convert correctly (e.g. "We assume that the wage offer sequence is **IID**…" in [Chapter 1](https://quantecon.github.io/book-dp1/ch-intro), previously lowercase "iid"), and cross-references inside table cells are covered by a converter regression test.

The **Common Symbols table itself** is a hand-curated page, so its fixes are queued separately in QuantEcon/book-dp1#351 and not yet applied — that covers the indicator `𝟙` (a KaTeX limitation with `\mathbb{1}`; we'll map it to a renderable glyph), the missing `M(X)` cross-reference to §2.3.3.4, small-caps `IID` in the table, and the editorial items ("and … respectively", "etc." vs "…") which are pending an author call.

So: nothing to re-validate on this page just yet — I'll ping here once #351 lands. (If you're re-checking the others, book-dp-public#25 and the ✅ items in book-dp-public#26 are live now.)

**Comment by mmcky** [comment]
As promised — the Common Symbols table fixes have now landed and are **live**: https://quantecon.github.io/book-dp1/common-symbols

What changed (QuantEcon/book-dp1#351, commit QuantEcon/book-dp1@51a54ac):

- **Indicator** now renders with the double-struck look (𝟙-style) instead of a plain `1` — this applies everywhere `\1` appears in the book, not just the table. (Implementation note: KaTeX supports neither `\mathbb{1}` nor the raw `𝟙` glyph, so it uses the standard overlapped-strokes construction.)
- **`M(X)` row** has the missing cross-reference: "(see 2.3.3.4)", matching the PDF's §2.3.3.4.
- **`iid` → `IID`** (small caps in the PDF).
- **Editorial alignment with the PDF**: "ℕ, ℤ, ℝ and ℂ … respectively", and "etc." instead of "…".
- The `D(F)`/`L(X)`/`M(X)` script-font shape difference remains — that one is a genuine KaTeX-vs-LaTeX font difference we've accepted as a known limitation.

@Chihiro2000GitHub if you could validate these against the PDF when you get a chance, that would be super helpful — with these, all three of your reports (#24, #25, #26's converter items) are deployed. Thanks again!

**Comment by Chihiro2000GitHub** [comment]
Many thanks for the update, @mmcky!

I checked the live version against the PDF, and these changes look good to me now.

I'll move on to checking the changes you made in other issues and will report back soon.

Thanks again!

**Comment by mmcky** [comment]
I am looking at the latest site and I still see some rendering differences on the live site. 

In text: Chapter 2

<img width="728" height="158" alt="Image" src="https://github.com/user-attachments/assets/55247fd0-3147-4e23-8dd8-0f105d95e3b5" />

and in Common Symbols table

<img width="728" height="47" alt="Image" src="https://github.com/user-attachments/assets/e9369e07-82a6-4f32-a3a2-92998bae5558" />

These should be the same symbols right? 

(cc: @Chihiro2000GitHub @jstac)

**Comment by Chihiro2000GitHub** [comment]
Thanks, @mmcky! (cc: @jstac)

Yes, I can also confirm that there is a difference in the rendering of `M`.

My understanding is that the earlier accepted limitation was about the font difference between the PDF and the HTML version. But this case seems different, since the discrepancy is within the HTML version itself. In that sense, I agree that these should be the same symbols.

I also checked whether this might be a more general issue with `\mathscr` notation in the Common Symbols table. 

`L(X)` seems to show a similar discrepancy: the symbol in the Common Symbols table appears different from the corresponding notation in the Chapter 2 text in the HTML version.

In text: Chapter 2 (in the HTML)

<img width="517" height="65" alt="Image" src="https://github.com/user-attachments/assets/c4d1c3cd-3771-4f56-91d0-ed6e35e54825" />

and in Common Symbols table (in the HTML as well)

<img width="542" height="45" alt="Image" src="https://github.com/user-attachments/assets/bbc89a40-ca48-446a-92d0-1f52142f4638" />

On the other hand, I also checked `D(F)`, but I did not see the same discrepancy there.
The rendering in the Common Symbols table and in the main text seems consistent for `D`.

**Comment by mmcky** [comment]
@Chihiro2000GitHub this can be closed now right? Please confirm and then close when you're ready. 

<img width="1504" height="980" alt="Image" src="https://github.com/user-attachments/assets/70cd447e-2efe-43b7-a696-bc5176bf3e7f" />

**Comment by Chihiro2000GitHub** [comment]
@mmcky -- Yes, the comparison looks good to me. The rendering now appears consistent.

I think this can be closed now. Thank you!
